HHS/Head-Start-TTADP
# [TTAHUB-3199] New TR email alerts

## Description of change
- Added new emails based on the alerts (criterion is the same)
- Updated language in existing TR emails

## How to test
Confirm that [the emails](https://docs.google.com/document/d/1FsUqBxr5lqJ4iCjlRhpzZ9Z7Gtl9-0dBVkGWAszBPI8/edit), updated and added, are delivered and the verbiage is correct.

Note that you will need to adjust the schedule used in the cron.js to test the daily emails. You will also obviously need to force TRs/sessions in particular states to get the correct deliverables.

## Issue(s)
* https://jira.acf.gov/browse/TTAHUB-3199

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
